test/e2e/framework/framework.go

```diff
@@ -80,6 +83,14 @@ func New(kubeConfigPath string) (*Framework, cleanUpFunc, error) {
 		return nil, nil, errors.Wrap(err, "creating kubeClient failed")
 	}
 
+	jsonClientConfig := rest.CopyConfig(config)
```

Comment:
we already create `openshiftRouteClient` just below, why not use it?

Reply:
Good point, addressed.